### PR TITLE
Entity dialogs: Remove paper element and align dialog header style

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Align the header style on desktop more with default material dialog styles as we use throughout the app.

Replace `paper-tabs` and `app-toolbar` with `LitElement` versions.

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-header-bar.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+} from "lit-element";
+// @ts-ignore
+import topAppBarStyles from "@material/top-app-bar/dist/mdc.top-app-bar.min.css";
+
+@customElement("ha-header-bar")
+export class HaHeaderBar extends LitElement {
+  @property({ type: Boolean }) centerTitle = false;
+
+  protected render() {
+    let title = html`<span class="mdc-top-app-bar__title">
```
Comment (balloob) [inline src/components/ha-header-bar.ts:17]: Let's make this a ternary statement.

Comment (balloob) [reply src/components/ha-header-bar.ts:17]: or if…else.

File: src/components/ha-header-bar.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      </div>
+    </header>`;
+  }
+
+  static get styles() {
+    return [
+      unsafeCSS(topAppBarStyles),
+      css`
+        .mdc-top-app-bar {
+          position: static;
```
Comment (balloob) [inline src/components/ha-header-bar.ts:54]: Are we sure we want to put this in the element itself and not have the parent decide on this?

Comment (balloob) [reply src/components/ha-header-bar.ts:54]: This could be used outside of dialogs, no?

**Review by balloob — APPROVED** [review]

File: src/components/ha-header-bar.ts (hunk 1 above)
Comment (bramkragten) [reply src/components/ha-header-bar.ts:17]: the second one uses the first one.

File: src/components/ha-header-bar.ts (hunk 2 above)
Comment (bramkragten) [reply src/components/ha-header-bar.ts:54]: If you would want to use on the top scope you should use `<mwc-top-app-bar>`

File: src/components/ha-header-bar.ts (hunk 1 above)
Comment (bramkragten) [reply src/components/ha-header-bar.ts:17]: But I ended up not using it because it doesn't work well, so might as well remove it...

**Pull request merged**